.travis.yml

```diff
@@ -33,3 +33,6 @@ after_success:
 cache:
   directories:
     - $HOME/.glide/cache
+
+before_cache:
+  - find $HOME/.glide/cache -name ORIG_HEAD -exec rm {} \;
```

Comment:
Awesome optimisation! Might be a good idea to leave a comment explaining what it's doing